### PR TITLE
Update example code in timeseries_dataset.py

### PR DESCRIPTION
**Pull request by SuryanarayanaY**
I have gone through the example-2 in the TF documentation of [keras.utils.timeseries_dataset_from_array](https://www.tensorflow.org/api_docs/python/tf/keras/utils/timeseries_dataset_from_array). With existing code we can only generate one batch of dataset.In TF forum one of the user raised concern that users may confuse why we are using `input_data = data[:-10]` as it can generate only one batch and there will be loss of data and users may confuse `-10` as `sequence_length` here. Though the intention in the example is to demo on generating batches of data using the API,Its better to use `input_data = data[:]` to avoid confusion and also it give correct demo to generate the total possible no of batches without loss of data.

Also I am adding the `sequence_length` as variable in the code to avoid hard coding in the data indexing which will make it better understandable.

Thanks!

### CODE REVIEW

File: tf_keras/utils/timeseries_dataset.py [hunk 1]
```diff
@@ -110,16 +110,23 @@ def timeseries_dataset_from_array(
     timesteps to predict the next timestep, you would use:
 
     ```python
-    input_data = data[:-10]
-    targets = data[10:]
+    data = tf.range(15)
+    sequence_length =10
```
Comment (grasskin) [inline tf_keras/utils/timeseries_dataset.py:114]: Make sure to run the formatter (add a space before 10)

**Review by grasskin — CHANGES_REQUESTED** [review]
I think overall it's a helpful change - thank you for the PR!

**Comment by gbaned** [comment]
Hi @SuryanarayanaY Can you please check @grasskin's comments ? Thank you!

**Comment by SuryanarayanaY** [comment]
Done the changes for black formatting.Please review.

**Review by grasskin — APPROVED** [review]
LGTM, thank you!
